src/filtergraph.c

```diff
@@ -0,0 +1,138 @@
+#include <stdlib.h>
+#include <stdbool.h>
+
+#include <libavcodec/avcodec.h>
+#include <libavfilter/avfilter.h>
+#include <libavfilter/buffersink.h>
+#include <libavfilter/buffersrc.h>
+#include <libavformat/avformat.h>
+
+#include "libvx.h"
```

Comment:
Long time size I coded c/c++ :) But I was wondering about these includes... if they should be in the header file instead size the function definitions there use types defined in some of these h-files?

Reply:
There are different conventions, and a lot of different viewpoints on this topic! I will leave it as it is for now, but follow up after I have done some more research.